package.json

```diff
@@ -161,11 +162,10 @@
     "debounce-stream": "^2.0.0",
     "deep-freeze-strict": "1.1.1",
     "end-of-stream": "^1.4.4",
-    "eth-block-tracker": "^5.0.1",
+    "eth-block-tracker": "^6.0.0",
     "eth-ens-namehash": "^2.0.8",
     "eth-json-rpc-filters": "^4.2.1",
-    "eth-json-rpc-infura": "^5.1.0",
```

Comment:
We are bumping this finally 😅 I have updated `createInfuraClient` to account for the change in usage (default import -> named import).

Reply:
This is the most impactful change in this PR as `@metamask/eth-json-rpc-infura` 6.0.0 bumped `eth-json-rpc-middleware`, `eth-rpc-errors`, and `json-rpc-engine` ([see changelog entry](https://github.com/MetaMask/eth-json-rpc-infura/blob/main/CHANGELOG.md#600---2022-05-04)). I have gone over the changelogs for these and there don't appear to be any changes that we need to worry about but I probably need a second set of eyes on this 👀

Reply:
For `eth-json-rpc-middleware`, the change in version 8 that adds `eth_signTransaction` stood out to me. We should test this to ensure this is blocked. I'm pretty sure that it is blocked in our method middleware but haven't manually teseted.

Reply:
This all seems good to me!

Reply:
Ah, okay. I'll check this then!